### PR TITLE
Code generation example

### PR DESCRIPTION
**Pull request by hsiang-wu**
An example of how we can use `sympy. autowrap` to automatically generate C code and wrap it in Python.

### CODE REVIEW

**Review by CactusWin — COMMENTED** [review]
Quick Q: What will the codegen be used for?

File: neuralpp/test/quick_tests/symbolic/normalizer_test.py [hunk 1]
```diff
@@ -282,3 +282,41 @@ def test_quantifier_normalizer_integration():
                      if_then_else(B > 4, product2, product3),
                      ))
     assert normalizer.normalize(expr, Z3SolverExpression.from_expression(A) & (B > 4)).syntactic_eq(product)
+
+
+def test_codegen():
+    from sympy.utilities.codegen import codegen
```
[the remark this answers is not in the document]
Comment (hsiang-wu) [reply neuralpp/test/quick_tests/symbolic/normalizer_test.py:288]: I feel these are more "local-ish"?

**Comment by hsiang-wu** [comment]
> Quick Q: What will the codegen be used for?

Generating formula in C code which should run faster than natively in Python, so we have a more competitive run time and compelling story

**Review by rodrigodesalvobraz — APPROVED** [review]
Great!

**Pull request merged**